### PR TITLE
Update some links and add some staff

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: _posts/1970-01-01-architecture.md [hunk 1]
```diff
@@ -156,9 +156,8 @@ TODO: ROM example
 
 Is a collection of next-generation Ruby libraries, each intended to encapsulate a common task. Decoupled and reusable.
 
-TODO: Add extended dry-rb gems description
```
Comment (iJackUA) [inline _posts/1970-01-01-architecture.md:159]: Please, can you leave this TODO, as I really want to put here a list of brief descriptions and examples for most notable dry gems (as I see them): validation, types, container, auto_inject, component, configurable.

**Comment by iJackUA** [comment]
Thanks :) After this https://github.com/iJackUA/rwdtow/pull/1#discussion_r64534294 I'll merge it

**Comment by davydovanton** [comment]
@iJackUA done 😃

**Comment by davydovanton** [comment]
thanks for merge! 💚

**Comment by iJackUA** [comment]
Celebrating first PR in this repo 🎉 
Hopefully in one week I'll find time to finish some TODOs.

**Pull request merged**